clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -151,13 +161,34 @@ func (ste *simpleTestExecutor) ExecutePhase(ctx Context, phase *api.Phase) []err
 }
 
 // ExecuteObject executes single test object operation based on provided object configuration.
-func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32) []error {
+func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32, operation OperationType) []error {
+	var errList []error
+	template, err := config.ReadTemplate(object.ObjectTemplatePath)
+	if err != nil {
+		return []error{fmt.Errorf("reading template (%v) error: %v", object.ObjectTemplatePath, err)}
+	}
+	gvk := template.GroupVersionKind()
+
 	if namespace == "" {
 		// TODO: handle cluster level object
 	} else {
-		// TODO: handle namespaces level object
+		objName := fmt.Sprintf("%v-%d", object.Basename, replicaIndex)
+		switch operation {
+		case CREATE_OBJECT:
+			if err := ctx.GetFramework().CreateObject(namespace, objName, template); err != nil {
+				errList = append(errList, fmt.Errorf("namespace %v object %v creation error: %v", namespace, objName, err))
```

Comment:
instead of recomputing the name string everytime in each error msg, could you just compute once at top and use that?

Reply:
I could, however only one of those cases will be used at the time and errors (should) rarely happen. If it's computed at the top, in most cases it will not be even used. I prefer to keep it as it is.

Reply:
You're right. It's computed only once, so it's ok.